packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md

```diff
@@ -7,6 +7,8 @@ category: management
 
 <ApiLifecycle access="ea" />
 
+> **Note:** The MyAccount API is being enhanced, accessible at `/api/v1/idp/myaccount`. Use this endpoint if you plan to use the MyAccount API. The existing endpoint (`/api/v1/myaccount`) is deprecated.
```

Comment:
The version has been moved from the path to the `Accept` header, the value should be `application/json; okta-version=1.0.0`. For now the only valid version is `1.0.0`